Resources/Prototypes/_SCP/Entities/Clothing/OuterClothing/labcoats.yml

```diff
@@ -0,0 +1,83 @@
+﻿- type: entity
+  parent: ClothingOuterStorageFoldableBase
+  id: SCPClothingOuterCoatLabSci
+  name: foundation researcher's lab coat
+  description: A suit that protects against minor chemical spills. It has a purple stripe on the shoulder and purple trim on the sleeves, denoting it as a SCP Foundation labcoat.
+  components:
+  - type: Sprite
+    sprite: _SCP/Clothing/OuterClothing/Coats/scilabcoat.rsi
+  - type: Clothing
+    sprite: _SCP/Clothing/OuterClothing/Coats/scilabcoat.rsi
+  - type: Armor
+    modifiers:
+      coefficients:
+        Caustic: 0.75
+
+- type: entity
+  parent: [ClothingOuterStorageFoldableBaseOpened, SCPClothingOuterCoatLabSci]
+  id: SCPClothingOuterCoatLabSciOpened
+  name: foundation researcher's lab coat
+
+- type: entity
+  parent: ClothingOuterStorageFoldableBase
+  id: SCPClothingOuterCoatLabAsst
```

Comment:
remove reduntant clothing

Reply:
This was meant to be different than the RD coat... I had just broken the rd coat rsi file previously and never readded it.